### PR TITLE
Create ui-c8y-1021-9-0-limits-tab-in-tenant-details-7260-graft-release-cd-7669.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by pawel-rynarzewski-sag

Ticket: [MTM-61090](https://cumulocity.atlassian.net/browse/MTM-61090)
Original PR: [7669](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7669)

[MTM-61090]: https://cumulocity.atlassian.net/browse/MTM-61090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1021-9-0-limits-tab-in-tenant-details-7260-graft-release-cd-7669.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Limits" tab in tenant details (#7260) [GRAFT][release/cd] (#7669)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-9-0-limits-tab-in-tenant-details-7260-graft-release-cd-7669.md]: The Claude AI suggested to use the following title:
```suggestion
title: Added "Limits" tab to tenant details
```

File: content/change-logs/platform-services/ui-c8y-1021-9-0-limits-tab-in-tenant-details-7260-graft-release-cd-7669.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61090
+version: 1021.9.0
+---
+"Limits" tab in tenant details (#7260) [GRAFT][release/cd] (#7669)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1021-9-0-limits-tab-in-tenant-details-7260-graft-release-cd-7669.md]: The Claude AI suggested to use the following body:
```suggestion
To provide a better overview of the resource limits and quotas configured for a tenant, a new "Limits" tab has been added to the tenant details page. This tab allows to configure the maximum number of devices, requests, and queue sizes for the tenant. With this change, tenant administrators can easily view and manage the resource limits for their subtenant in the dedicated tab.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@pawel-rynarzewski-sag Please review and approve

**Review by pawel-rynarzewski-c8y — APPROVED** [review]

**Review by darlucumulo — APPROVED** [review]

**Review by kowa-c8y — APPROVED** [review]

**Pull request merged**